File: c-sharp/test/HelloWorldTest/HelloTest.cs
using System;
using Xunit;
using HelloWorld;

namespace HelloWorldTest
{
	public class HelloTest
	{

		[Fact]
		public void DefaultMessage() {
			Assert.True(
				Hello.SayHello().Contains(Hello.defaultUser)
			);
		}

		[Fact]
		public void CustomUsername() {
			String user = "Me";
			Assert.True(
				Hello.SayHello(user).Contains(user)
			);
		}


	}
}

File: java/src/test/java/z596/helloworld/HelloTest.java
package z594.helloworld;

import org.junit.Test;
import org.junit.Assert;
import org.hamcrest.CoreMatchers;

public class HelloTest {

	@Test
	public void defaultMessage() {
		Assert.assertThat(
			Hello.sayHello(),
			CoreMatchers.containsString(Hello.defaultUser)
		);
	}

	@Test
	public void customUsername() {
		String user = "Me";
		Assert.assertThat(
			Hello.sayHello(user),
			CoreMatchers.containsString(user)
		);
	}

}
